pkg/controller/mobiledeveloperconsole/mobiledeveloperconsole.go

```diff
@@ -240,8 +240,8 @@ func newMDCDeploymentConfig(cr *mdcv1alpha1.MobileDeveloperConsole) (*openshifta
 							},
 							Resources: corev1.ResourceRequirements{
 								Limits: corev1.ResourceList{
-									corev1.ResourceLimitsMemory: resource.MustParse("128Gi"),
-									corev1.ResourceLimitsCPU:    resource.MustParse("60m"),
+									corev1.ResourceMemory: resource.MustParse("128Gi"),
```

Comment:
is this a typo? the limit is set to 128G for memory?

Reply:
👍